### DISCUSSION TITLE
Google auth [Jan 25]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4995] [Discord Threads]
> 🙋‍♂️ **TheBigBoss**  
#### nocodb version: 0.90.7  
![image0](https://cdn.discordapp.com/attachments/661905455894888493/968159104579301466/unknown.png)  
![image1](https://cdn.discordapp.com/attachments/661905455894888493/968159104826736730/unknown.png)

**Comment by o1lab** [comment]
> 💁 **o1lab**  
#### Please raise an issue with the details  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/968159105745293373)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
